src/node/extension/extension.ts

```diff
@@ -388,15 +386,31 @@ function startSession(config: any): StartSessionResult {
 		}
 	}
 
+	let determineDebugTypeP: Promise<string|null>;
+	if (config.request === 'attach' && typeof config.processId === 'string' && config.processId.trim() === '${command:PickProcess}') {
```

Comment:
The check for '${command:PickProcess}' shows the problematic aspect of this PR: instead of letting VS Code handle the variable substitution generically, the node-debug extension now does the substitution explicitly, before VS Code gets a chance for doing it. But I don't think that this will break anybody... 
But may be we should consider to check for the alternative command `${command:extension.pickNodeProcess}` as well.